DiffAugment-stylegan2-pytorch/training/training_loop.py

```diff
@@ -256,11 +252,22 @@ def training_loop(
             stats_tfevents = tensorboard.SummaryWriter(run_dir)
         except ImportError as err:
             print('Skipping tfevents export:', err)
+        try:
+            import comet_ml
+        except ImportError as err:
+            comet_ml = None
+            print('Skippeing comet export: ', err)
 
     # Train.
     if rank == 0:
         print(f'Training for {total_kimg} kimg...')
         print()
+        experiment = comet_ml.ExistingExperiment(
```

Comment:
What in case, when we do not add api_key (e.g. for debugging)? Won't it be better to check it firstly?